analytics/management/commands/populate_analytics_db.py

```diff
@@ -76,13 +76,13 @@ def insert_fixture_data(stat, fixture_data, table):
         stat = COUNT_STATS['realm_active_humans::day']
         realm_data = {
             None: self.generate_fixture_data(stat, .1, .03, 3, .5, 3, partial_sum=True),
-        } # type: Dict[Optional[str], List[int]]
+        } # type: Mapping[Optional[str], List[int]]
         insert_fixture_data(stat, realm_data, RealmCount)
         FillState.objects.create(property=stat.property, end_time=last_end_time,
                                  state=FillState.DONE)
 
         stat = COUNT_STATS['messages_sent:is_bot:hour']
-        user_data = {'false': self.generate_fixture_data(stat, 2, 1, 1.5, .6, 8, holiday_rate=.1)}
+        user_data = {'false': self.generate_fixture_data(stat, 2, 1, 1.5, .6, 8, holiday_rate=.1)}  #type: Mapping[Optional[str], List[int]]
```

Comment:
This line is too long, and there's some missing whitespace here.   Otherwise looks good.